packages/ckeditor5-engine/src/view/domconverter.ts

```diff
@@ -1168,8 +1168,12 @@ export default class DomConverter {
 		// we will use the fact that range will collapse if it's end is before it's start.
 		const range = this._domDocument.createRange();
 
-		range.setStart( selection.anchorNode!, selection.anchorOffset );
-		range.setEnd( selection.focusNode!, selection.focusOffset );
+		try {
+			range.setStart( selection.anchorNode!, selection.anchorOffset );
+			range.setEnd( selection.focusNode!, selection.focusOffset );
+		} catch {
+			return false;
```

Comment:
Probably you should call `range.detach();` before.

Reply:
Ok, ignore me, it does nothing https://developer.mozilla.org/en-US/docs/Web/API/Range/detach